src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        setColor(color);
+        this.radius = radius;

```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for the color and radius and pass them on for better code readability.